core/src/main/scala/org/genivi/sota/core/Boot.scala

```diff
@@ -83,6 +82,10 @@ object Boot extends App with DatabaseConfig {
     case _ => {
       val notifier = DefaultUpdateNotifier
       Http().bindAndHandle(routes(notifier), host, port)
```

Comment:
@jerrytrieu starting webservice api and vehicles api right? strip webservice of vehicles route? atm we would have duplicated routes for vehicles when in non-rvi mode